.gitignore

```diff
@@ -6,4 +6,3 @@ test/z-ensure-coverage.js
 
 *.*.swp
 dist/
-examples/stage-2-shield-study/addon/StudyUtils.jsm
```

Comment:
This was moved to a [.gitignore](https://github.com/mozilla/shield-studies-addon-utils/pull/78/files#diff-2054ac9a8ccbc631331888139d01b03c) within the examples folder [as per our previous discussion](https://github.com/mozilla/shield-studies-addon-utils/pull/77#discussion_r128598424).